src/pages/order-service/index.tsx

```diff
@@ -18,7 +18,8 @@ export type Equipment = {
   tippingNumber: string
   serialNumber: string
   type: string
-  status: string
+  situação: string
```

Comment:
Vou te pedir uma mudança que vai dar um pouco de trabalho, mas quero até a opinião do outros EPS (@JoaoVitorFarias @SamButers @AlineLermen @AlvaroLeles @RuanMoura ) sobre isso:

Quando estiver escrevendo código e for dar nome para variáveis, ou nessas ocasiões de `chave: valor`, não usar acentuações nos nomes das variáveis nem das "chaves".

Então esse código e os outros parecidos nos outros arquivos ficariam assim:
```suggestion
  situacao: string
```

É comum não usarmos acentuações, até porque pode dar erro ou conflito de leitura de arquivo etc a depender do computador, tipo se ele lê em UTF 8 etc.

Reply:
Alterações feitas.